pkg/detectors/quickbase/quickbase_test.go

```diff
@@ -0,0 +1,118 @@
+	package quickbase
```

Comment:
Please add the tags back. You can run the test by `go test ./pkg/detectors/quickbase -tags=detectors`
```
//go:build detectors
// +build detectors
```